internal/releasesjson/downloader.go

```diff
@@ -226,6 +227,7 @@ var licenseFiles = []string{
 	"EULA.txt",
 	"TermsOfEvaluation.txt",
 	"LICENSE.txt",
+	"LICENSE",
```

Comment:
While we do recognise this file name when _building_ products I would be hesitant to recognising it here as well. The long-term intentions are that all products are _packaged_ with `LICENSE.txt` only.

Whether this will be reflected in the source repository or if it gets renamed as part of a release is a separate concern (hence the note about "best effort" in the comment there).

I would assume/hope that all names will eventually align with `LICENSE.txt`. I know it may feel weird but the context here is that license files were historically _always_ part of the repo but only recently are being added to the archives, so there is shorter history and hence we can be more restrictive here and avoid the tech debt.